templates/home.html

```diff
@@ -98,7 +98,7 @@ <h6 >Time Remaining: [[timeLeft]]</h6>
                     Play the daily prompt to start/continue your streak! 🔥
                 </div>
                 <div class="card-footer text-muted" v-else>
-                    Log in to play this daily prompt and start your streak! 🔥
+                    Log in and play the daily prompt and start your streak! 🔥
```

Comment:
nit: This sentence has too many "and"s now and sounds kind of clunky, I'd suggest changing it to 

`Log in to play the daily prompt and start your streak! 🔥`

or

`Log in and play the daily prompt to start your streak! 🔥`

but I prefer the first one

Reply:
Added